### PR TITLE
OS-31: Basic Syntax

### PR DESCRIPTION
**Pull request by shayaanmiyy**
Jira Ticket URL: https://digitalpolygon.atlassian.net/browse/OS-31

- Learn Twig syntax: variables, filters, functions, and control structures.

- Understand template inheritance and include functionality.

### Variables and Filters:

- Create a Twig template that defines and displays variables.

- Use various Twig filters to manipulate and display data.

### Control Structures:

- Implement conditional statements (if, else) and loops (for) in a Twig template.

- Create a template that loops through a list of items and displays them.

### Template Inheritance:

- Set up a base template and extend it with child templates.

- Use the block and extends tags to create a consistent layout across multiple pages.

###

image

Result ScreenShot

### CODE REVIEW

File: web/app/themes/dp-timber-theme/index.php [hunk 1]
```diff
@@ -16,7 +16,8 @@
 $context          = Timber::context();
 $context['posts'] = Timber::get_posts();
 $context['foo'] = 'Shayaan';
-$templates        = array( 'index.twig' );
+$context['qux'] = 'lorem ipsum doller si amut  ';
+$templates        = array( 'variables.twig' );
```
Comment (Prashant114606) [inline web/app/themes/dp-timber-theme/index.php]: add partials and include it inside the index.twig

File: web/app/themes/dp-timber-theme/index.php [hunk 2]
```diff
@@ -16,7 +16,8 @@
 $context          = Timber::context();
 $context['posts'] = Timber::get_posts();
 $context['foo'] = 'Shayaan';
-$templates        = array( 'index.twig' );
+$context['qux'] = 'lorem ipsum doller si amut  ';
```
Comment (Prashant114606) [inline web/app/themes/dp-timber-theme/index.php]: what's the purpose of this varibale?

Comment (shayaanmiyy) [reply web/app/themes/dp-timber-theme/index.php]: I will change the variable name to be more descriptive

File: web/app/themes/dp-timber-theme/index.php (hunk 1 above)
Comment (shayaanmiyy) [reply web/app/themes/dp-timber-theme/index.php]: Yes I created a basic.twig file in Partials and included it in Index.twig using 

`{% include "partial/basic.twig" %}`

Comment (shayaanmiyy) [reply web/app/themes/dp-timber-theme/index.php]: I've updated the code, please review it

File: web/app/themes/dp-timber-theme/index.php (hunk 2 above)
Comment (shayaanmiyy) [reply web/app/themes/dp-timber-theme/index.php]: I've updated the code, please review it

**Pull request merged**